### DISCUSSION TITLE
CBU Tuya chip

### DISCUSSION THREAD

**Discussion by giangi007** [arendst/Tasmota#13097] [Q&A] [answered]
Hello to All, I bought this Tuya IR Remote control from aliexpress https://it.aliexpress.com/item/1005003032192623.html?spm=a2g0s.9042311.0.0.36604c4dt7YF46 and the chip inside is CBU (https://developer.tuya.com/en/docs/iot/cbu-module-datasheet?id=Ka07pykl5dk4u). Is it possible to flash tasmota firmware?

Thanks

**Comment by s-hadinger** [comment, marked as answer]
No unfortunately

**Reply by barbudor** [reply]
It should be still possible to perform a brain transplant and use a tasmota compatible module based on ESP8266 or ESP32/ESP32-C3/ESP32-S2

I would assume that for a IR blaster like this one you don't need many pins, so may be a ESP-01M,  ESP-M03 or even the classis ESP12E/F would be suitable.
They are not pin to pin compatible but a some how a "flying" method can work are done recently by a user on Discord : ![image](https://user-images.githubusercontent.com/2996042/132703632-941e279d-474a-46f0-851e-b9e0e3f6a9f2.png)

You can get more help on Tasmota's Discord, channel #hardware-talk
